lib/alex_koin/slack_commands.ex

```diff
@@ -43,6 +64,14 @@ defmodule AlexKoin.SlackCommands do
     coin
   end
 
+  def transfer(from_wallet, to_wallet, amount, memo) do
+    # 1. get coins to transfer
+    coins = Coin |> Ecto.Query.where(wallet_id: ^from_wallet.id) |> Ecto.Query.limit(^amount) |> Repo.all
```

Comment:
Please multiline when there's more than 1 pipe

Reply:
Also, please separate this query into coin.ex. Coin is your reference to the database. Functions that make queries for those "models" are a great place to put queries.

Not needed, but personally I like using query syntax directly. Something in coin.ex like:

```
def get_coin_by_wallet_query(wallet_id, amount = 10) do
  query = 
    from c in "coins",
    where: c.wallet_id = ^wallet_id,
    limit: ^amount
end
```

Then call it here like
```
coins = 
  Coin.get_coin_by_wallet_query(from_wallet.id, amount)
    |> Repo.all
```

Choose whatever syntax you like, but the main point is to put queries and database specific things in the "models" and allow contexts/modules to compose queries using them.